### PR TITLE
Add tenure profile badges to `Project Loved Team`

### PR DESCRIPTION
**Pull request by minusQuantumNeko**
## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/People/Project_Loved_Team/es.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+outdated_translation: true
```
Comment (cl8n) [inline wiki/People/Project_Loved_Team/es.md:2]: `outdated_since` as well

File: wiki/People/Project_Loved_Team/en.md [hunk 2]
```diff
@@ -26,6 +26,13 @@ Some team additions may be added manually on a case-by-case basis in order to fi
 
 The criteria for joining the team vastly differs from role to role, but proficiency and community presence is always sought after.
 
+## Benefits
+
+Project Loved Team members are granted the following benefits:
+
+- **Tenure badge**: A [profile badge](/wiki/Community/Profile_badge) that displays the number of years they have been in the LVD, ranging from 1 to 6.
```
Comment (cl8n) [inline wiki/People/Project_Loved_Team/en.md:33]:
```suggestion
- **Tenure badge**: A [profile badge](/wiki/Community/Profile_badge) that displays the number of years they have been helping run Project Loved.
```

File: wiki/People/Project_Loved_Team/en.md [hunk 3]
```diff
@@ -26,6 +26,13 @@ Some team additions may be added manually on a case-by-case basis in order to fi
 
 The criteria for joining the team vastly differs from role to role, but proficiency and community presence is always sought after.
 
+## Benefits
+
+Project Loved Team members are granted the following benefits:
```
Comment (cl8n) [inline wiki/People/Project_Loved_Team/en.md:31]: "Project Loved Team members are given profile badges that display the number of years [...]"
no need for 1-item list

**Review by cl8n — CHANGES_REQUESTED** [review]
feels cringe to present this as a "benefit" but I guess w/e cuz other pages say the same thing

also the "5 years" note on the last line of the article is wrong now. that whole last sentence can just be removed

**Comment by minusQuantumNeko** [comment]
I do agree it's not the best to present it as a benefit, but yes, I used the same thing other teams pages use

File: wiki/People/Project_Loved_Team/en.md [hunk 4]
```diff
@@ -26,6 +26,13 @@ Some team additions may be added manually on a case-by-case basis in order to fi
...
+
+- **Tenure badge**: A [profile badge](/wiki/Community/Profile_badge) that displays the number of years they have been helping run Project Loved.
```
Comment (minusQuantumNeko) [inline wiki/People/Project_Loved_Team/en.md:33]: Maybe this?
```suggestion
Project Loved Team members are given a tenure [profile badge](/wiki/Community/Profile_badge) that displays the number of years they have been helping run Project Loved.
```

**Review by cl8n — APPROVED** [review]

**Pull request merged**